build_library/dev_image_util.sh

```diff
@@ -96,6 +96,7 @@ create_dev_image() {
 
   set_image_profile dev
   emerge_to_image "${root_fs_dir}" @system ${base_pkg}
+  run_ldconfig "${root_fs_dir}"
```

Comment:
@marineam We call emerge_to_image twice.  First in start_image for sys-apps/baselayout, and later for the ${base_pkg}. For the start_image call ldconfig is not installed yet, so run_ldconfig will fail for arm64 there.